src/event_handlers/circlesBacking.ts

```diff
@@ -0,0 +1,42 @@
+import { CirclesBacking } from "generated";
+
+CirclesBacking.CirclesBackingInitiated.handler(async ({ event, context }) => {
+  context.CirclesBacking.set({
+    id: event.transaction.hash,
+    blockNumber: event.block.number,
+    timestamp: event.block.timestamp,
+    transactionIndex: event.transaction.transactionIndex,
+    logIndex: event.logIndex,
+    transactionHash: event.transaction.hash,
+    circlesBackingInstance: event.params.circlesBackingInstance,
+    personalCirclesAddress: event.params.personalCirclesAddress,
+    backer_id: event.params.backer,
+    backingAsset: event.params.backingAsset,
+    lbpAddress: "",
+  });
+});
+
+CirclesBacking.CirclesBackingCompleted.handler(async ({ event, context }) => {
+  const circlesBacking = await context.CirclesBacking.get(
+    event.transaction.hash
+  );
+
+  // If the circles backing is not found, return
+  if (!circlesBacking) {
+    return;
+  }
+
+  context.CirclesBacking.set({
+    id: event.transaction.hash,
+    blockNumber: event.block.number,
+    timestamp: event.block.timestamp,
+    transactionIndex: event.transaction.transactionIndex,
+    logIndex: event.logIndex,
+    transactionHash: event.transaction.hash,
```

Comment:
your are overriding the parameters from `CirclesBackingInitiated`. is this on purpose?

Reply:
Hm yea youre right, i would like to keep both. 
So I will create in the schema maybe:

```
 timestampInitiated 
vs
timestampCompleted
``` etc etc